nav2_collision_monitor/params/collision_monitor_params.yaml

```diff
@@ -14,6 +14,7 @@ collision_monitor:
     # Footprint could be "polygon" type with dynamically set footprint from footprint_topic
     # or "circle" type with static footprint set by radius. "footprint_topic" parameter
     # to be ignored in circular case.
+    # or "velocity_polygon" type with dynamically set polygon from velocity_polygons
```

Comment:
Adjust this comment

> Footprint could be "polygon" type with  ...

to be numbered (1) (2) (3) for the 3 options instead of tacking on the extra "or" fragment